xml/System/GCMemoryInfo.xml

```diff
@@ -316,7 +316,7 @@ The memory after `OBJ_D` is not considered part of the `FragmentedBytes` but is
         <summary>Gets the physical memory load when the last garbage collection occurred.</summary>
         <value>The physical memory load, in bytes, when the last garbage collection occurred.</value>
         <remarks>
-          <para>On Windows, memory load is a field in the [MEMORYSTATUS structure](/windows/win32/api/winbase/ns-winbase-memorystatus). It's a number between 0 and 100 that specifies the approximate percentage of physical memory that's in use (0 indicates no memory use and 100 indicates full memory use). Corresponding information is given on other operating systems.</para>
+          <para>On Windows, memory load is a field in the [MEMORYSTATUS structure](https://learn.microsoft.com/en-us/windows/win32/api/winbase/ns-winbase-memorystatus) in bytes that denotes the physical memory that's in use if the process isn't running in a container with a memory limit. For other operating systems, or if the process is running on Windows in a container with a memory limit, memory load is either the cgroup's physical memory in use or the current process's working set.</para>
```

Comment:
```suggestion
          <para>On Windows, memory load is a field in the [MEMORYSTATUS structure](/windows/win32/api/winbase/ns-winbase-memorystatus) in bytes that denotes the physical memory that's in use if the process isn't running in a container with a memory limit. For other operating systems, or if the process is running on Windows in a container with a memory limit, memory load is either the cgroup's physical memory in use or the current process's working set.</para>
```

Reply:
Thanks for the feedback and the link! Seems like that link was broken on the msdn page: https://learn.microsoft.com/en-us/dotnet/api/system.gcmemoryinfo.memoryloadbytes?view=net-8.0#remarks. Will be happy to confirm that the link worked once this is out but thought I'd still mention why I changed this line.

Reply:
Oh I see, it's using a Markdown link format in a non-Markdown section. I updated my suggestion.